sapper/src/components/Navigation/Nav.svelte

```diff
@@ -18,36 +18,42 @@
   ]
 </script>
 
-<nav class="px-8 flex justify-between lg:max-w-screen-xl mx-auto">
-  <a
-    class="py-5"
-    aria-current={segment === undefined ? 'page' : undefined}
-    href=".">
-    <img src="iteam.png" alt="Iteam" class="logo" style="width: 100px;" />
-  </a>
+<div class={!segment && 'bg-grey-cod text-white'}>
+  <nav class="px-8  flex justify-between lg:max-w-screen-xl mx-auto">
+    <a
+      class="py-5"
+      aria-current={segment === undefined ? 'page' : undefined}
+      href=".">
+      <img
+        src={!segment ? 'iteam-white.png' : 'iteam.png'}
+        alt="Iteam"
+        class="logo"
+        style="width: 100px;" />
```

Comment:
nothing majorly important, but could use tailwind here. w-24 = 6rem = 96px
```suggestion
      <img
        src={!segment ? 'iteam-white.png' : 'iteam.png'}
        alt="Iteam"
        class="logo w-24" />
```